internal/web/assets/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG BASE_IMAGE=ubuntu:20.04
```

Comment:
base image is 20.04 but we dont offer that on the webui. If this is an obligatory build-arg, should we remove the default?